### ISSUE TITLE
No module named celery.five

### ISSUE THREAD

**Issue by qiaocci** [celery/django-celery-results#157]
Celery 5.x remove the `celery.five` and drop support for python2:
``` 
 File "/home/qiaocc/.venv/devops_env/lib/python3.8/site-packages/django_celery_results/models.py", line 9, in <module>
    from celery.five import python_2_unicode_compatible
ModuleNotFoundError: No module named 'celery.five'
```
 `django-celery-results` can also drop the support for python2

**Comment by thedrow** [comment]
We haven't touched the django integrations yet.
PRs are welocme to expedite this.

**Comment by ZviBaratz** [comment]
This should also be resolved by #163.
